frmts/gtiff/geotiff.cpp

```diff
@@ -17385,6 +17415,18 @@ CPLErr GTiffDataset::OpenOffset(TIFF *hTIFFIn, toff_t nDirOffsetIn,
                         m_fJXLDistance = static_cast<float>(dfVal);
                     }
                 }
+                else if (m_nCompression == COMPRESSION_JXL &&
+                         EQUAL(pszKey, "JXL_ALPHA_DISTANCE"))
+                {
+                    const double dfVal = CPLAtof(pszValue);
+                    if (dfVal > 0 && dfVal <= 15)
+                    {
+                        m_oGTiffMDMD.SetMetadataItem(
+                            "COMPRESSION_REVERSIBILITY", "LOSSY",
+                            "IMAGE_STRUCTURE");
+                        m_fJXLAlphaDistance = static_cast<float>(dfVal);
+                    }
```

Comment:
Does it make sense to report an out of bounds warning here?

Reply:
The generic logic that parses creation options inspects the value regarding the 'min' and 'max' value and will emit a warning.
Here it is a bit different as it parses metadata registered by GDAL itself, so it should be valid, unless someone has voluntarily corrupted it. I don't think there's a need for an extra check here